app/models/autofinger.rb

```diff
@@ -3,12 +3,11 @@ class Autofinger < ActiveRecord::Base
   self.primary_keys = :owner, :interest
   self.table_name = :autofinger
   validates_presence_of :owner, :interest, :priority
+  validates :priority, inclusion: { in: [0, 1, 2, 3] }
 
   belongs_to :interested_party, foreign_key: :owner, class_name: 'Account'
   belongs_to :subject_of_interest, foreign_key: :interest, class_name: 'Account'
 
-  validates_presence_of :interest, :owner
```

Comment:
Umm... for some reason it doesn't show in this snippet, but if you view the whole file, you'll see it.